src/client/components/UserHeader.js

```diff
@@ -83,7 +84,12 @@ const UserHeader = ({
             )}
           </div>
           <div className="UserHeader__handle-rank-container">
-            <div className="UserHeader__row UserHeader__handle">@{handle}</div>
+            <div className="UserHeader__row UserHeader__handle">
+              @{handle}&nbsp;
```

Comment:
Can we use `{' '}` instead of `&nbsp;`?

Reply:
okay.

Reply:
the `{' '}` didn't work as space. returned to `&nbsp;`

Reply:
Why cant you just use ``{`@${handle}   ` }`` ?

Reply:
oh yeah @Sekhmet that can work too

Reply:
> Why cant you just use {`@${handle} ` }

I tried that too. The space gets trimmed.

Reply:
> I feel like those could be separated using margin, instead of hard coded space.

Apply a style class separately for the `@handle` and `follows you` text?

Reply:
Currently, the code follows the format `@handle follows you`.
On which style can the margin be applied?

Code:
```
            <div className="UserHeader__row UserHeader__handle">
              @{handle}&nbsp;
              {!isSameUser && isFollowingYou && (
                <FormattedMessage id="follows_you" defaultMessage="follows you" />
              )}
            </div>
```

Reply:
Put the 'follows you' in a `span`?

Reply:
you can do `.UserHeader__handle > span { margin-left: 5px; }` this targets the first span that gets generated by `<FormattedMessage/>` components